drivers/soft_spi/Kconfig

```diff
@@ -11,3 +11,7 @@ config MODULE_SOFT_SPI
     depends on TEST_KCONFIG
     select MODULE_PERIPH_GPIO
     select MODULE_XTIMER
+
+config MODULE_SOFT_SPI_AS_PERIPH_SPI
+    bool "Software-implemented SPI implements periph/spi API"
+    select MODULE_SOFT_SPI
```

Comment:
Where did that come from? :wink:

Reply:
Ups was looking at @maribu soft-spi PR at the same time...removed